tested on cloud-vm and got this error (`docker logs connect`)

```bash
===> User
uid=1000(appuser) gid=1000(appuser) groups=1000(appuser)
===> Configuring ...
===> Running preflight checks ... 
===> Check if Kafka is healthy ...
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/usr/share/java/cp-base-new/log4j-slf4j-impl-2.24.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/usr/share/java/cp-base-new/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
Error while running kafka-ready.
org.apache.kafka.common.KafkaException: Failed to create new KafkaAdminClient
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:569) ~[kafka-clients-8.0.3-ccs.jar:?]
        at org.apache.kafka.clients.admin.Admin.create(Admin.java:143) ~[kafka-clients-8.0.3-ccs.jar:?]
        at org.apache.kafka.clients.admin.AdminClient.create(AdminClient.java:49) ~[kafka-clients-8.0.3-ccs.jar:?]
        at io.confluent.admin.utils.ClusterStatus.isKafkaReady(ClusterStatus.java:55) ~[utility-belt-8.0.3-26.jar:?]
        at io.confluent.admin.utils.cli.KafkaReadyCommand.main(KafkaReadyCommand.java:133) [utility-belt-8.0.3-26.jar:?]
Caused by: org.apache.kafka.common.config.ConfigException: No resolvable bootstrap urls given in bootstrap.servers
        at org.apache.kafka.clients.ClientUtils.parseAndValidateAddresses(ClientUtils.java:104) ~[kafka-clients-8.0.3-ccs.jar:?]
        at org.apache.kafka.clients.ClientUtils.parseAndValidateAddresses(ClientUtils.java:63) ~[kafka-clients-8.0.3-ccs.jar:?]
        at org.apache.kafka.clients.admin.internals.AdminBootstrapAddresses.fromConfig(AdminBootstrapAddresses.java:72) ~[kafka-clients-8.0.3-ccs.jar:?]
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:533) ~[kafka-clients-8.0.3-ccs.jar:?]
        ... 4 more
Using log4j config /etc/cp-base-new/log4j2.yaml
```

**Update**: seems to be a hickup, after `docker compose restart` the issue was gone.
Testing with local spring-boot application calling the vm for connect and mariadb worked as intended.

--- a/uc-witness-protection/connectors/Exercise0jdbc-connector.json
+++ b/uc-witness-protection/connectors/Exercise0jdbc-connector.json
@@ -9,7 +9,7 @@
     "value.converter": "org.apache.kafka.connect.storage.StringConverter",
     "key.converter": "org.apache.kafka.connect.storage.StringConverter",
     "mode":"bulk",
-    "table.blacklist":"none",
+    "table.whitelist":"events",
     "poll.interval.ms" : 3600
   }
 }

